org.github.Powder-Toy.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: org.github.Powder-Toy

```

Comment:
The app-id is wrong. You should either use `com.github.ThePowderToy.ThePowderToy` or ask upstream what app-id they want to use.

Reply:
Fixed.